modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/configuring-pci-passthrough.adoc
+
+[id="virt-adding-kernel-arguments-to-enable-iommu-for-the-bios_{context}"]
+= Adding kernel arguments to enable IOMMU for the BIOS
+
+To enable an IOMMU extension for the BIOS (Basic Input/Output System) associated with a PCI device, create the `MachineConfig` object file and add the kernel arguments for the device.
+
```

Comment:
I just want to clarify a couple of things:
 - enabling VT-d in the BIOS is a prerequisite for PCI passthrough.
 - IOMMU kernel argument needs to be configured once per node, not per device.

I think we should say something like

```
To enable the IOMMU driver in the kernel, create the `MachineConfig` object file and add the kernel arguments.
```

Reply:
Updated the content.